esti/lakectl_local_test.go

```diff
@@ -0,0 +1,101 @@
+package esti
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/treeverse/lakefs/pkg/local"
+)
+
+func createTestData(t *testing.T, vars map[string]string, objects []string) {
+	for _, o := range objects {
+		vars["FILE_PATH"] = o
+		RunCmdAndVerifySuccessWithFile(t, Lakectl()+" fs upload -s files/ro_1k lakefs://"+vars["REPO"]+"/"+vars["REF"]+"/"+vars["FILE_PATH"], false, "lakectl_fs_upload", vars)
+	}
+	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" commit lakefs://"+vars["REPO"]+"/"+vars["REF"]+" --allow-empty-message -m \" \"", false, "lakectl_commit_with_empty_msg_flag", vars)
+}
+
+func TestLakectlLocal_clone(t *testing.T) {
+	tmpDir := t.TempDir()
+	_, err := os.CreateTemp(tmpDir, "")
```

Comment:
close the file handle or use alternative way to create a temporary file

Reply:
Yup, sorry 😅